### DISCUSSION TITLE
Ablity to have more then one commandFile

### DISCUSSION THREAD

**Discussion by Pierzety** [eritislami/evobot#256] [Q&A]
Is it possible to have more then one commandFile if you wish to integrate more features into the bot and you don't want to cram it all in one folder. For example I wish to find source code so I can also add moderation commands and I do not want to be restricted to just one file for everything.

const commandFiles = readdirSync(join(__dirname, "misc" )).filter((file) => file.endsWith(".js"));
      ^
SyntaxError: Identifier 'commandFiles' has already been declared

**Comment by eja0iq** [comment]
well im using this in my index.js
`const load = dirs => {
  const commands = readdirSync(join(__dirname, "commands", `/${dirs}`)).filter(d => d.endsWith('.js'));
  for (let file of commands) {
      let pull = require(join(__dirname, "commands", `/${dirs}/${file}`));
      client.commands.set(pull.name, pull);
  };
};
["name of file example: Music", "other file"].forEach(x => load(x));`

**Comment by Pierzety** [comment]
So I would replace what I posted below here that what you used?

const commandFiles = readdirSync(join(__dirname, "commands")).filter((file) => file.endsWith(".js"));
for (const file of commandFiles) {
  const command = require(join(__dirname, "commands", `${file}`));
  client.commands.set(command.name, command);
}
